packages/combobox/docs/listboxData.js

```diff
@@ -0,0 +1,65 @@
+export const listboxData = [
```

Comment:
Copy from packages/listbox. We were importing it from listbox before but we don't have an export maps entry for it so WDS could not resolve it. Imo, just copying is fine for this...